src/ol/expr/cpu.js

```diff
@@ -281,6 +296,26 @@ function compileAccessorExpression(expression, context) {
   }
 }
 
+/**
+ * @param {Array<ExpressionEvaluator>} argEvaluators Argument evaluators
+ * @param {function(Array): ReturnType} evaluator Final evaluator taking in the evaluated args
+ * @return {function(EvaluationContext):ReturnType|UNKNOWN} the evaluator function; if any arg evaluated to UNKNOWN, will return UNKNOWN
+ * @template ReturnType
+ */
+function checkForUnknown(argEvaluators, evaluator) {
```

Comment:
reading this again after a while, I find this function very confusing; I'll try to rework this logic to make it less obtuse for a new eye